hkmc2/shared/src/main/scala/hkmc2/codegen/HandlerLowering.scala

```diff
@@ -398,7 +400,9 @@ class HandlerLowering(paths: HandlerPaths, opt: EffectHandlers)(using TL, Raise,
 
     val PartRet(head, states) = go(blk)(using labelIds, N)
     
-    val (headState, restStates) = optParts(BlockState(0, head, N), states)
+    // optParts doesn't seem to do anything, but we'll keep it just in case it's needed in the future
```

Comment:
So the `optParts` function is currently dead code? In that case, I'd prefer to comment it as well, so it does not become a needless maintenance burden.